platform/kube/inject/inject.go

```diff
@@ -40,7 +40,9 @@ import (
 // Defaults values for injecting istio proxy into kubernetes
 // resources.
 const (
+	DefaultHubEnvVar       = "MANAGER_HUB"
```

Comment:
FWIW, the current cmd/istioctl/inject.go is _very_ kubernetes specific (i.e. `kube-inject` and not `inject`).